### PR TITLE
Update user accounts filtertable

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

**Comment by driusan** [comment]
@nicolasbrossard Can you test this? The code looks reasonable to me..

File: modules/user_accounts/jsx/columnFormatter.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function formatColumn(column, cell, rowData) {
+    if(column === 'Username') {
+        var url = loris.BaseURL + "/user_accounts/edit_user/?identifier=" + rowData[1];
```
Comment (xlecours) [inline modules/user_accounts/jsx/columnFormatter.js]: You could add rowHeaders in the formatColumn function's signature and then add the mapping function. This way the hardCoded index at `rowData[1]` could be changed for `row.Username`

see: https://github.com/aces/Loris/pull/1950

**Comment by stellarxo** [comment]
Implement Xavier's change and see Samir to be merged

File: modules/user_accounts/jsx/columnFormatter.js (hunk 1 above)
Comment (kongtiaowang) [inline modules/user_accounts/jsx/columnFormatter.js]: @xlecours I fixed them, thanks.

**Comment by stellarxo** [comment]
image

**Comment by nirtiac** [comment]
Also tested; I get the same error as Stella.

**Pull request closed without merging**